ui/App/UserProfileScreen.svelte

```diff
@@ -30,14 +43,105 @@
       },
     });
   }
+
+  let registration: Registration | undefined;
+  let ownedOrgs: Org[] = [];
+
+  const wallet = svelteStore.get(Wallet.store);
+  const ethereumEnvironment = ethereum.selectedEnvironment;
+
+  async function loadSidebarData(): Promise<void> {
+    const address = user.metadata.ethereum?.address;
+    console.log(user);
+    if (!address) {
+      return;
+    }
+
+    try {
```

Comment:
I think we don't need this extra error handling here. If there's an exception, it will be caught by the top-level error handler and shown to the user. So the whole `try/catch` can be removed.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/2559/commits/609defab41cd8bb1efa0932c158dc954f9a87ab4

Reply:
Where is this error handled?

Reply:
Also wouldn't I also remove the try catch in the `ProfileScreen > loadSidebarData()`

Reply:
All errors that are not explicitly handled are caught here at the top level:
https://github.com/radicle-dev/radicle-upstream/blob/master/ui/App.svelte#L63.

Reply:
And yeah, let's remove it also from `ProfileScreen > loadSidebarData()`.